api/src/main/resources/org/openmrs/api/db/hibernate/OrderSet.hbm.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+    <class name="OrderSet" table="order_set" batch-size="25">
+
+        <cache usage="read-write"/>
+
+        <id name="orderSetId" type="java.lang.Integer"
+            column="order_set_id"
+            unsaved-value="undefined">
+            <generator class="org.openmrs.api.db.hibernate.NativeIfNotAssignedIdentityGenerator" />
+        </id>
+
+        <property name="name" type="java.lang.String"
+                  column="name" not-null="true" />
+        <property name="description" type="java.lang.String"
+                  column="description" not-null="true" />
+        <property name="operator" column="operator" not-null="true">
+            <type name="org.openmrs.util.HibernateEnumType">
```

Comment:
Should I change it right now. Because I see all the 1.12.x versions uses **org.openmrs.util.HibernateEnumType**

Changing this to **org.hibernate.type.EnumType** 
gives me this error while running tests
**Unable to instantiate custom type: org.hibernate.type.EnumType**